Dockerfile

```diff
@@ -0,0 +1,85 @@
+ARG DOCKER_NAMESPACE
+ARG VERSION_RUST_CONTAINER
+
+ARG DISTROLESS_IMAGE=gcr.io/distroless/base-debian12:nonroot
+FROM ${DISTROLESS_IMAGE} AS distroless
+
+FROM ${DOCKER_NAMESPACE}/rust:${VERSION_RUST_CONTAINER} AS builder
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    ca-certificates \
+    cmake \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+ARG TARGETARCH
+
+WORKDIR "/work"
+
+COPY --from=distroless /var/lib/dpkg/status.d /distroless_pkgs
+
+COPY src src
+COPY Cargo.lock Cargo.lock
+COPY Cargo.toml Cargo.toml
+
+RUN cargo build --release --locked --target-dir ./build
+
+# replace by the following as soon as bookworm is suppoted
+# curl -s https://packagecloud.io/install/repositories/FZambia/centrifugo/script.deb.sh | bash
+RUN curl -LJO https://github.com/centrifugal/centrifugo/releases/download/v5.3.2/centrifugo_5.3.2_linux_${TARGETARCH}.tar.gz && \
+    tar -xzf centrifugo_*.tar.gz
+
+SHELL ["/bin/bash", "-c"]
+RUN <<EOT
+    set -eu
+
+    mkdir -p /copy/status.d
+
+    executable=(build/release/omnect-ui)
```

Comment:
do we have to take care of centrifugo here, as well?

Reply:
no. it's tested with Dockerfile already.